### ISSUE TITLE
Focus indicator for items inside top navigation panel

### ISSUE THREAD

**Issue by zqian** [tl-its-umich-edu/remote-office-hours-queue#754]
The focus indicator for the top menu items (Home, Resources & Support, What's New, and the user settings) does not have sufficient contrast with the background (is 1.6:1, should be at least 3:1).
